svelte-kit-scss/src/lib/components/shared/Dropdown/item-templates/DropdownItemTemplateCheckbox.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
+  import { Check16 } from 'svelte-octicons';
+
+  export let checked = false;
+  export let label: string;
+</script>
+
+<div class="dropdown-item-template-checkbox" class:checked>
```

Comment:
I wonder if we need to add some aria [`listbox` attributes](https://www.w3.org/TR/wai-aria-1.1/#listbox) to this custom multiselect dropdown